framework/src/application/node/node.ts

```diff
@@ -50,16 +52,135 @@ import { EventPostTransactionData } from '../../types';
 import { InMemoryChannel } from '../../controller/channels';
 import { EventInfoObject } from '../../controller/event';
 import { ApplicationState } from '../application_state';
+import { accountAssetSchema, defaultAccountAsset } from './account';
+import {
+	EVENT_PROCESSOR_BRADCASRT_BLOCK,
+	EVENT_PROCESSOR_SYNC_REQUIRED,
+} from './processor/processor';
+import { EVENT_SYNCHRONIZER_SYNC_RQUIRED } from './synchronizer/base_synchronizer';
 
 const forgeInterval = 1000;
 const { EVENT_NEW_BLOCK, EVENT_DELETE_BLOCK } = chainEvents;
 const { EVENT_ROUND_CHANGED } = dposConstants;
 const { EVENT_TRANSACTION_REMOVED } = txPoolEvents;
 
-export interface GenesisBlockInstance extends GenesisBlockJSON {
+interface Payload {
+	id: string;
+	senderPublicKey: string;
+	nonce: string;
+	fee: string;
+	signatures: string[];
+}
+
+interface TransferPayload extends Payload {
+	type: 8;
```

Comment:
can we required the transactions and use `TransferTransaction.Type`?

Reply:
we can, but this will be removed next week. this is temporal solution